File: Lista03_HerancaEPolimorfismo/PegadaCarbono/app/src/main/java/PegadaCarbono/Bicicleta.java
package PegadaCarbono;

public class Bicicleta implements PegadaDeCarbono{
    private String marca;
    private double aro; 
    private double calibragem; 

    public Bicicleta(String marca, double aro, double calibragem) {
        this.marca = marca;
        this.aro = aro;
        this.calibragem = calibragem;
    }

    public String getMarca() {
        return marca;
    }

    public double getAro() {
        return aro;
    }

    public double getCalibragem() {
        return calibragem;
    }

    public void setCalibragem(double calibragem) {
        this.calibragem = calibragem;
    }

    public double getPegadaCarbono(){
        return 0;
    }

}

File: Lista11_Python/Ex03_PegadaDeCarbono/bicicleta.py
class Bicicleta:

    def __init__(self, marca, aro, calibragem):
        self.marca = marca;
        self.aro = aro;
        self.calibragem = calibragem;

    def __str__(self):
        string = "Marca: "+self.marca+"\nAro: "+str(self.aro)+"\nCalibragem: "+str(self.calibragem)
        return string

    def getMarca(self):
        return self.marca

    def getAro(self):
        return self.aro
    
    def getCalibragem(self):
        return self.calibragem

    def getPegadaCarbono(self):
        pegCarb = 0
        return pegCarb
